lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -684,30 +684,27 @@ def add_stored_credential_options(xml, options = {})
       end
 
       def add_stored_credential_using_normalized_fields(xml, options)
-        if options[:stored_credential][:initial_transaction]
-          xml.storedCredentials 'usage' => 'FIRST'
-        else
-          reason = case options[:stored_credential][:reason_type]
-                   when 'installment' then 'INSTALMENT'
-                   when 'recurring' then 'RECURRING'
-                   when 'unscheduled' then 'UNSCHEDULED'
-                   end
-
-          xml.storedCredentials 'usage' => 'USED', 'merchantInitiatedReason' => reason do
-            xml.schemeTransactionIdentifier options[:stored_credential][:network_transaction_id] if options[:stored_credential][:network_transaction_id]
-          end
+        reason = case options[:stored_credential][:reason_type]
```

Comment:
The previous code did not include this if it was the `initial_transaction`, did we want to preserve that logic?

Reply:
No, Worldpay's docs are pretty specific about how to send the standing instruction reason on the first transaction, and this difference between their docs and our implementation is what spawned this ticket in the first place.